modules/tensor_mechanics/test/tests/pressure/tests

```diff
@@ -16,5 +16,23 @@
     issues = '#4781 #8235'
     design = 'syntax/BCs/Pressure/index.md'
     requirement = 'The Pressure boundary condition action shall create the objects needed to apply pressure boundary conditions on a 3D model as demonstrated by correctly computing the response of an elastic small-strain isotropic unit cube with pressure applied on three faces to create a hydrostatic pressure using the volumetric locking correction b-bar formulation.'
-  [../]
+  []
+  [jacobian_3D]
+    type = 'PetscJacobianTester'
+    input = 'cantilever.i'
+    run_sim = true
+    difference_tol = 5e-3
+    issues = '#19657'
+    design = 'Pressure.md'
+    requirement = "The Pressure boundary condition shall compute the correct Jacobian for a problem in 3D space."
+  []
```

Comment:
Do you verify the Jacobian for different element types?

Reply:
I'm adding a couple of tests that you'll see in the next set of changes.